src/Topics/TexasTransfers.hs

```diff
@@ -207,68 +207,76 @@ transferSlamInviteAccepted = let
 
 transferOverInterference :: Situations
 transferOverInterference = let
-    sit (overcall, transferBid) = let
+    sit (overcall, transferBid, suit) = let
         action = do
             setOpener T.North
             B.b1N
-            overcall
+            _ <- overcall
+            -- Performance optimization: have exactly a 6-card suit to make
+            -- these boards generate much faster
+            suitLength suit 6
         explanation =
             "Partner opened " .+ T.Bid 1 T.Notrump .+ ", and the next " .+
             "player interfered. As long as they only bid at the 2 or 3 " .+
             "level, Texas transfers are still on!"
         in situation "int" action transferBid explanation
   in
-    wrapNW $ return sit <~ [ (Nat.b1No2C,  B.b1N4D)
-                           , (Nat.b1No2D,  B.b1N4D)
+    wrapNW $ return sit <~ [ (Nat.b1No2C,  B.b1N4D, T.Hearts)
+                           , (Nat.b1No2D,  B.b1N4D, T.Hearts)
```

Comment:
No, put this in b1N4D and b1N4H directly. If it's sometimes a problem for this Situation, it's sometimes a problem for all of them. and it's a _much_ smaller change in the code.